modules/distr-tracing-rn-new-features.adoc

```diff
@@ -63,7 +89,7 @@ This release of {DTProductName} addresses Common Vulnerabilities and Exposures (
 
 This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
 
-With this release, the {JaegerName} Operator is now installed to the `openshift-distributed-tracing` namespace by default.  Previously the default installation had been in the `openshift-operator` namespace.
+With this release, the {JaegerName} Operator is now installed to the `openshift-distributed-tracing` namespace by default. Previously the default installation had been in the `openshift-operator` namespace.
```

Comment:
If somebody removes a Jaeger Operator running in `openshift-operators`, let Jaeger instances running and then installs a new version of the Jaeger Operator in `openshift-distributed-tracing`, the Jaeger instances will not be upgraded. Should we document that?

Reply:
It does not seem like a common use-case to uninstall the operator while there are running instances.

Reply:
I don't even know if it's possible to remove an Operator with instances still running.

Reply:
@JStickler it is.

Reply:
@iblancasa That feels like a usability bug to me.